libs/designsystem/src/lib/components/page/page.component.scss

```
@@ -24,12 +24,20 @@ ion-toolbar {
   --padding-bottom: 0;
 
   box-sizing: border-box;
-  padding-inline: utils.size('xs');
-  transition: padding-inline utils.get-transition-easing('enter-exit')
-    utils.get-transition-duration('short');
+
+  $button-padding: utils.size('xs');
+
+  padding-left: $button-padding;
+  padding-right: $button-padding;
+  transition-property: padding-left, padding-right;
+  transition-duration: utils.get-transition-duration('short');
+  transition-timing-function: utils.get-transition-easing('enter-exit');
 
   &.content-scrolled {
-    padding-inline: 0;
+    $button-padding: utils.size('xxxs');
+
+    padding-left: $button-padding;
+    padding-right: $button-padding;
```

If you use a CSS custom property instead you can control the padding with a single line:

```suggestion
    --button-padding: utils.size('xxxs');
```